src/index.js

```diff
@@ -77,7 +69,7 @@ function loadCollections(collectionsDir) {
 
 async function buildSchema() {
   const { lianaOptions, Implementation } = configStore;
-  const models = getModels();
+  const models = modelsManager.getModelArray();
```

Comment:
`getModelArray`seems weird IMO.

If I understand correctly, `modelsManager.getModels()` reads `models` in an object, and `getModelArray()`converts it to an array ?

Reply:
Exactly ;) the only place where we want array is here !